core/templates/domain/exploration/InteractionObjectFactorySpec.ts

```diff
@@ -771,4 +773,28 @@ describe('Interaction object factory', () => {
     expect(Interaction.getCustomizationArgContentIds(ca)).toEqual(
       ['ca_dummyCustArg_content_0', 'ca_dummyCustArg_content_1']);
   });
+
+  it('should fully cover constructing customization arguments for all' +
+     'interactions', () => {
+    Object.keys(INTERACTION_SPECS).forEach(interactionId => {
+      expect(() => {
+        const defaultCa = {};
+        const caSpecs = INTERACTION_SPECS[
+          interactionId].customization_arg_specs;
+        caSpecs.forEach(caSpec => {
+          defaultCa[caSpec.name] = caSpec.default_value;
+        });
+
+        iof.createFromBackendDict({
+          answer_groups: answerGroupsDict,
+          confirmed_unclassified_answers: [],
+          customization_args: defaultCa,
+          default_outcome: defaultOutcomeDict,
+          hints: hintsDict,
+          id: interactionId,
+          solution: solutionDict
+        });
+      }).not.toThrowError(`Unrecognized interaction id ${interactionId}`);
```

Comment:
Could you check that this doesn't throw any errors at all -- rather than the weaker check of ensuring it doesn't throw a particular one?